ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>Again, bad printing by invoking the variable. Also, <code>train_img_qty</code>  variable is created only to to show its value, why not just </p><p><code>print('Number of training examples: {%some_expression%}')</code>  ?</p><p> </p><p>Also, this way of subsetting a dataframe may return a series of a dataframe depending on other conditions, which is bad in general.</p><p>Consider an alternative for the expression:</p><p><code>(labels_df.is_valid.values).sum()</code>  and <code>(~labels_df.is_valid.values).sum()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>